docs/source/instrument/_install_new_instrument.rst

```diff
@@ -291,20 +295,94 @@ option:
 Start version control
 ---------------------
 
-While this step is optional, it is **highly recommended** that you place
+While this step is optional, it is **highly recommended** that you keep
 your bluesky instrument directory under some form of software version
 control. At minimum, this can provide some form of backup protection. It
 also helps others to collaborate with similar bluesky instruments by
 sharing your instrument's implementations.
 
-Instructions for using `git <https://git-scm.com/>`__ as software
-version control with `GitHub <https://github.com/>`__ or the `APS GitLab
-server <https://git.aps.anl.gov/>`__ are provided in `this separate
-document <https://bcda-aps.github.io/bluesky_training/reference/_github_create_repo.html>`__.
+The installer program initializes a local git repository in the ``bluesky``
+folder. We now need to create a blank remote repository, for example
+on `GitHub <https://github.com/>`_ or the `APS GitLab server
+<https://git.aps.anl.gov/>`_ using your `beamline organization 
+<https://bcda-aps.github.io/bluesky_training/reference/_git-help.html#beamline-github-organizations>`_. 
+Note that the bluesky team recommends GitHub (why is that? See 
```

Comment:
Similar suggestion to above.  For links from .rst document to content in the same or different .rst document, use Sphinx-style links, as shown in the next branch.  These links do not require the document name.  The default text is the section title, otherwise override as the examples show.

Reply:
See PR #221